src/components/TheResults.vue

```diff
@@ -10,6 +10,13 @@
                     :retail-price="
                         product.defaultVariant.currentVersion.retailPrice
                     "
+                    :image-path="
```

Comment:
that looks a bit horrible
maybe you can create a function for this?

Reply:
This only returns an empty string whenever the array is empty, isn't there a nice Typescript feature that does that more concisely?

Reply:
ts provides next to no helper functions
however there is a pay to do what you want, however it is debatable if this is really an improvement: `productVariant.currentVersion.medias.nodes[0]?.path ?? ""`

Reply:
I have rewritten it like that, does not seem worth it to put this one-liner in a function (: